outputs.tf

```diff
@@ -1,11 +1,3 @@
-output "id" {
```

Comment:
Commit message says "Remove not needed variables". Those are not variables, those are `outputs`. Please mention in the commit message, that this is because the underlying resources (`template_dir.bootstrap-manifests`) will be removed.